services/utils.js

```diff
@@ -0,0 +1,7 @@
+
+const bcrypt = require('bcrypt');
+
+module.exports.hashData = async data => {
+  const salt = await bcrypt.genSalt(10);
+  return await bcrypt.hash(data, salt);
+}
```

Comment:
Please, add blank line in the end of a file.

Reply:
+